website/content/api-docs/api-structure.mdx

```diff
@@ -40,9 +40,9 @@ $ curl \
     http://127.0.0.1:8500/v1/agent/members
 ```
 
-Previously this was provided via a `?token=` query parameter. This functionality
-exists on many endpoints for backwards compatibility, but its use is **highly
+Previously this was provided via a `?token=` query parameter but its use is **highly
 discouraged**, since it can show up in access logs as part of the URL.
+This functionality is deprecated and will be removed in future versions.
```

Comment:
Following up on @cthain 's comment from a previous PR...

Perhaps we should give explicit indication of when this will be removed. I don't expect all users to be familiar with our deprecation / removal policy.

Should we say something like:
```suggestion
Though the token could also be provided via a `?token=` query parameter,
that method is **highly discouraged** because the token can show up in access logs as part of the URL.
The `?token=` query parameter is deprecated and will be removed in Consul 1.17.
```